scripts/unlock-package-lock.js

```diff
@@ -20,13 +20,17 @@ function removeDependencies(dependencyName, packages) {
 
 function unlock(filename) {
   const packageLock = require(filename);
-  Object.keys(packageLock.packages).forEach(dependencyName => {
-    removeDependencies(dependencyName, packageLock.packages);
-  });
+  if (packageLock.packages) {
+    Object.keys(packageLock.packages).forEach(dependencyName => {
+      removeDependencies(dependencyName, packageLock.packages);
+    });
+  }
 
-  Object.keys(packageLock.dependencies).forEach(dependencyName => {
-    removeDependencies(dependencyName, packageLock.dependencies);
-  });
+  if (packageLock.dependencies) {
```

Comment:
Not sure what changes in the package lock that these may not be there. It affects the nested package locks which will be removed in the future anyway. This script will then be updated as well to be in sync with what we run in the other repos.